### PR TITLE
Release notes for 5.8

### PR DESCRIPTION
**Pull request by dennisdoomen**

### CODE REVIEW

File: docs/_pages/releases.md [hunk 1]
```diff
@@ -7,6 +7,21 @@ sidebar:
   nav: "sidebar"
 ---
 
+## 5.8.0
+
+**What's New**
+* Added thread-safety to tests using `AssertionScope` while running many tests in parallel [#1091](https://github.com/fluentassertions/fluentassertions/pull/1091)
+* Allow users to specify a custom `IAssertionStrategy` for the assertion scope, for instance, to create screenshots when a test fails - [#1094](https://github.com/fluentassertions/fluentassertions/pull/1094) & [#906](https://github.com/fluentassertions/fluentassertions/issues/906). 
+* Supports .NET Core Preview 7 - [#1107](https://github.com/fluentassertions/fluentassertions/pull/1107)
```
Comment (jnyrup) [inline docs/_pages/releases.md]: .NET Core *3.0*

File: docs/_pages/releases.md [hunk 2]
```diff
@@ -7,6 +7,21 @@ sidebar:
   nav: "sidebar"
 ---
 
+## 5.8.0
+
+**What's New**
+* Added thread-safety to tests using `AssertionScope` while running many tests in parallel [#1091](https://github.com/fluentassertions/fluentassertions/pull/1091)
```
Comment (jnyrup) [inline docs/_pages/releases.md]: Technically speaking it's when running many tests asynchronously

File: docs/_pages/releases.md [hunk 3]
```diff
@@ -7,6 +7,21 @@ sidebar:
...
+* Allow users to specify a custom `IAssertionStrategy` for the assertion scope, for instance, to create screenshots when a test fails - [#1094](https://github.com/fluentassertions/fluentassertions/pull/1094) & [#906](https://github.com/fluentassertions/fluentassertions/issues/906). 
+* Supports .NET Core Preview 7 - [#1107](https://github.com/fluentassertions/fluentassertions/pull/1107)
+
+**Fixes**
+* `Excluding` with `BeEquivalentTo` did not always work well with overridden properties - [#1087](https://github.com/fluentassertions/fluentassertions/pull/1087) & [#1077](https://github.com/fluentassertions/fluentassertions/issues/1077)
+* Failure message formatting threw a `FormatException` when a nested message contains braces - [#1092](https://github.com/fluentassertions/fluentassertions/pull/1097)
```
Comment (jnyrup) [inline docs/_pages/releases.md]: The link text is 1092 but the link is to 1097.

File: docs/_pages/releases.md [hunk 4]
```diff
@@ -7,6 +7,21 @@ sidebar:
...
+* Fixed confusing `(Not)BeAssignableTo` failure messages - [#1104](https://github.com/fluentassertions/fluentassertions/pull/1104) & [#1103](https://github.com/fluentassertions/fluentassertions/issues/1103)
+* Fixed a potential leakage of failure messages between multiple assertions - [#1105](https://github.com/fluentassertions/fluentassertions/pull/1105)
+
+Kudos to [@conklinb](https://github.com/conklinb) and [Amaury Leve](https://github.com/Evangelink) for notable contributions and my partner-in-crime [Jonas Nyrup](https://github.com/jnyrup) for some of the fixes, a lot of (internal) quality improvements and his critical eye.
```
Comment (jnyrup) [inline docs/_pages/releases.md]: Leve => Levé

Comment (jnyrup) [inline docs/_pages/releases.md]: @ conklinb => conklinb

File: docs/_pages/releases.md (hunk 2 above)
Comment (jnyrup) [inline docs/_pages/releases.md]: A missing `-` before the link to 1091

**Comment by dennisdoomen** [comment]
> his critical eye. 

You just proved my point ;-)

**Pull request merged**